ledger/metrics/src/main/scala/com/daml/metrics/IndexDBMetrics.scala

```diff
@@ -73,6 +73,45 @@ class IndexDBMetrics(override val prefix: MetricName, override val registry: Met
   val lookupContractByKeyDbMetrics: DatabaseMetrics = createDbMetrics(
     "lookup_contract_by_key"
   )
+
+  // TODO etq: Tune metric names
+  val flatTxIdsCreate: DatabaseMetrics = createDbMetrics(
+    "get_create_event_ids_for_flat_transactions"
+  )
+  val flatTxIdsConsuming: DatabaseMetrics = createDbMetrics(
+    "get_consuming_event_ids_for_flat_transactions"
+  )
+  val flatTxPayloadCreate: DatabaseMetrics = createDbMetrics(
+    "get_create_event_payloads_for_flat_transactions"
+  )
+  val flatTxPayloadConsuming: DatabaseMetrics = createDbMetrics(
+    "get_consuming_event_payloads_for_flat_transactions"
+  )
+  val treeTxIdsCreateStakeholder: DatabaseMetrics = createDbMetrics(
+    "get_create_event_ids_for_stakeholder_for_tree_transactions"
+  )
+  val treeTxIdsCreateNonStakeholderInformee: DatabaseMetrics = createDbMetrics(
+    "get_create_event_ids_for_nonstakeholderinformee_for_tree_transactions"
+  )
+  val treeTxIdsConsumingStakeholder: DatabaseMetrics = createDbMetrics(
+    "get_consuming_event_ids_for_stakeholder_for_tree_transactions"
+  )
+  val treeTxIdsConsumingNonStakeholderInformee: DatabaseMetrics = createDbMetrics(
+    "get_consuming_event_ids_for_nonstakeholderinformee_for_tree_transactions"
+  )
+  val treeTxIdsNonConsumingInformee: DatabaseMetrics = createDbMetrics(
+    "get_nonconsuming_event_ids_for_informee_for_tree_transactions"
+  )
+  val treeTxPayloadCreate: DatabaseMetrics = createDbMetrics(
+    "get_create_event_payloads_for_tree_transactions"
+  )
+  val treeTxPayloadConsuming: DatabaseMetrics = createDbMetrics(
+    "get_consuming_event_payloads_for_tree_transactions"
+  )
+  val treeTxPayloadNonConsuming: DatabaseMetrics = createDbMetrics(
```

Comment:
AFAIK, some metrics get obsolete with this PR e.g. `get_flat_transactions` etc
We need a note in the release notes for that

Reply:
I checked and it almost gets obsolete: it's no longer used for measuring queries but it's still used for measuring lf translation time `get_flat_transactions.translation`. The same applies to `get_transaction_trees`.

Added a TODO to review and document those changes